codecov.yml

```diff
@@ -0,0 +1,26 @@
+coverage:
+  status:
+    project:
+      default: false
+      core:
```

Comment:
These target percentages were based off of what current coverage percentages were.

Reply:
That's fine. We can adjust as we go...